models/models.js

```diff
@@ -68,4 +68,22 @@ exports.postNewComment = (inputId, comment) => {
     .then(({ rows }) => {
         return rows[0]
     })
+}
+
+exports.deleteCommentById = (inputId) => {
+    const id = [inputId]
+    return db.query(`DELETE FROM comments WHERE comment_id = $1 RETURNING *;`, id)
+    .then(({rows}) => {
+        if (!rows.length) {
+            return Promise.reject({ status: 404, msg: 'not found'})
+        }
+        return db.query(`SELECT * FROM comments WHERE comment_id = $1;`, id)
+    })
+    .then(({ rows }) => {
+        if (!rows.length) {
+            return rows
+        } else {
+            return 'unable to delete comment'
+        }
+    })
```

Comment:
This portion of the model isn't really needed.
Once we have a respond and we know the comment has been deleted we're all good.

Reply:
Sweet thank you